src/ParseUser.js

```diff
@@ -439,7 +439,7 @@ class ParseUser extends ParseObject {
    * @return {Promise} A promise that is fulfilled with the user when
    *     the login is complete.
    */
-  logIn(options?: FullOptions): Promise<ParseUser> {
+  logIn(options?: FullOptions, usePost?: boolean): Promise<ParseUser> {
```

Comment:
Can you move `usePost` to options? Also add it to documentation.

Reply:
Done. Thank you so much for the quick review!